addons/sale_stock/views/sale_order_views.xml

```diff
@@ -68,6 +71,13 @@
                 <xpath expr="//field[@name='team_id']" position="after">
                     <field name="warehouse_id" options="{'no_create': True}" groups="stock.group_stock_multi_warehouses" optional="hide"/>
                 </xpath>
+                <field name="invoice_status" position="before">
+                    <field name="effective_date" invisible="1"/>
+                    <field name="delivery_status" optional="hide" widget="badge"
+                        decoration-success="(commitment_date and commitment_date &gt; context_today().strftime('%Y-%m-%d 23:59:59')) or (delivery_status=='full' and commitment_date and effective_date &lt;= commitment_date)"
+                        decoration-danger="(commitment_date and commitment_date &lt; context_today().strftime('%Y-%m-%d 00:00:00')) or (delivery_status=='full' and commitment_date and effective_date &gt; commitment_date)"
+                        decoration-warning="commitment_date and commitment_date &gt; context_today().strftime('%Y-%m-%d 00:00:00') and commitment_date &lt; context_today().strftime('%Y-%m-%d 23:59:59') and delivery_status!='full'"/>
+                </field>
```

Comment:
Same decoration comments as the purchase_order_view (except with `commitment_date` instead of  `date_planned`